posts/views.py

```diff
@@ -1,3 +1,28 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from .models import Post
+from . import forms
+from django.utils.crypto import get_random_string
+
+def post_list(request):
+    posts = Post.objects.all()
+    return render(request,'posts/posts_list.html',{'posts':posts})
+
+def post(request,id):
+    post = Post.objects.get(id=id)
+    return render(request, 'posts/post.html', {'post': post})
+
+def create_post(request):
```

Comment:
Use a class based view instead of branching out for each methods. Use `get` and `post` methods now, use `dispatch` method for later, if you have logic that should run in common before each method.

Reply:
ohkay i will make this change and update soon